bufr2geojson/__init__.py

```diff
@@ -318,6 +424,103 @@ def get_location(self) -> Union[dict, None]:
             "coordinates": location
         }
 
+    def get_zcoordinate(self, BUFRclass=None):
```

Comment:
Annotate the type of `BUFRclass` (I would also rename to `bufr_class`).

Also add a return type.